docs/vcs.rst

```diff
@@ -340,3 +340,66 @@ monolingual translations).
 In the background Weblate creates Git repository for you and all changes are
 tracked in in. In case you decide later to use VCS to store the translations,
 it's already within Weblate and you can base on that.
+
+.. _vcs-gitlab:
+
+GitLab
+------
+
+.. versionadded:: 3.10
```

Comment:
I think we can do it for 3.9 :-)

Reply:
Ah, I didn't check the latest version. Will do.

Reply:
```suggestion
.. versionadded:: 3.9
```